app/models/work_package.rb

```diff
@@ -68,50 +68,58 @@ class WorkPackage < ActiveRecord::Base
   attr_protected :project_id, :author_id, :lft, :rgt
   # <<< issues.rb <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
 
-  scope :recently_updated, order: "#{WorkPackage.table_name}.updated_at DESC"
-  scope :visible, lambda {|*args|
-    { include: :project,
-      conditions: WorkPackage.visible_condition(args.first ||
-                                                                 User.current) }
+  scope :recently_updated, ->() {
+    # Specified as a String due to https://github.com/rails/rails/issues/15405
+    # TODO: change to Hash on upgrade to Rails 4.1.
+    order("#{WorkPackage.table_name}.updated_at DESC")
+  }
+
+  scope :visible, ->(*args) {
+    includes(:project)
+    .where(WorkPackage.visible_condition(args.first || User.current))
   }
 
   scope :in_status, -> (*args) do
                       where(status_id: (args.first.respond_to?(:id) ? args.first.id : args.first))
                     end
 
-  scope :for_projects, lambda { |projects|
-    { conditions: { project_id: projects } }
+  scope :for_projects, ->(projects) {
+    where(project_id: projects)
   }
 
-  scope :changed_since, lambda { |changed_since|
-    changed_since ? where(["#{WorkPackage.table_name}.updated_at >= ?", changed_since]) : nil
+  scope :changed_since, ->(changed_since) {
+    if changed_since
+      where(["#{WorkPackage.table_name}.updated_at >= ?", changed_since])
+    else
+      nil
+    end
   }
 
   # >>> issues.rb >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
-  scope :open, conditions: ["#{Status.table_name}.is_closed = ?", false], include: :status
+  scope :open, ->() {
+    includes(:status)
+    .where(statuses: { is_closed: false  })
```

Comment:
Unnecessary spacing detected.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.